### PR TITLE
ci(dependabot): setup dependabot

### PR DESCRIPTION
**Pull request by jorisre**

### CODE REVIEW

File: dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```
Comment (bluebill1049) [inline dependabot.yml:6]: `weekly`?

**Review by bluebill1049 — APPROVED** [review]
LGTM!

File: dependabot.yml (hunk 1 above)
Comment (jorisre) [reply dependabot.yml:6]: Just to start and I'll change it later

Comment (bluebill1049) [reply dependabot.yml:6]: kk 👍

**Pull request merged**